requirements.txt

```diff
@@ -1,8 +1,12 @@
-setuptools
+setuptools~=59.8.0
 numpy>=1.16.0
-pandas
-mdanalysis>=2.0.0b0
-pytest
-pathlib
-matplotlib
-scipy
\ No newline at end of file
+pandas~=1.2.4
+mdanalysis~=2.0.0.dev0
+pytest~=6.2.4
+pathlib~=1.0.1
+matplotlib~=3.4.2
+scipy~=1.7.0rc1
+yaml~=0.2.5
+pyyaml~=6.0
+versioneer~=0.20
+statsmodels~=0.12.2
```

Comment:
These are very specific — can't you relax the requirements? In particular, having a MDAnalysis beta in there is not good.

Reply:
Absolutely, thanks for the catch. Pycharm autogenerated a new requirements.txt file and I didn't catch it.